tailwind.config.js

```diff
@@ -1,4 +1,3 @@
-const { tint, lighten } = require('polished')
```

Comment:
Have you removed `polished` as a dependency? I guess it is not used somewhere else now, right?